internal/services/network/application_gateway_resource.go

```diff
@@ -2503,13 +2778,71 @@ func expandApplicationGatewayBackendHTTPSettings(d *pluginsdk.ResourceData, gate
 	return &results
 }
 
-func flattenApplicationGatewayBackendHTTPSettings(input *[]network.ApplicationGatewayBackendHTTPSettings) ([]interface{}, error) {
-	results := make([]interface{}, 0)
-	if input == nil {
-		return results, nil
-	}
+func expandApplicationGatewayBackendSettings(d *pluginsdk.ResourceData, gatewayID string) *[]network.ApplicationGatewayBackendSettings {
+	results := make([]network.ApplicationGatewayBackendSettings, 0)
+	vs := d.Get("backend_settings").(*schema.Set).List()
 
-	for _, v := range *input {
+	for _, raw := range vs {
+		v := raw.(map[string]interface{})
+
+		name := v["name"].(string)
+		port := int32(v["port"].(int))
+		protocol := v["protocol"].(string)
+		timeout := int32(v["timeout"].(int))
+
+		setting := network.ApplicationGatewayBackendSettings{
+			Name: &name,
+			ApplicationGatewayBackendSettingsPropertiesFormat: &network.ApplicationGatewayBackendSettingsPropertiesFormat{
+				PickHostNameFromBackendAddress: utils.Bool(false),
```

Comment:
Should this be exported to the schema?

Reply:
No, confirmed with service team that ture is not supported now.

Reply:
Create API issue